src/framework/components/collision/system.js

```diff
@@ -643,6 +645,14 @@ class CollisionComponentSystem extends ComponentSystem {
             data.halfExtents = new Vec3(data.halfExtents[0], data.halfExtents[1], data.halfExtents[2]);
         }
 
+        if (data.linearOffset && Array.isArray(data.linearOffset)) {
```

Comment:
don't really need this part `data.linearOffset &&` as isArray returns false for undefined or null parameter